lib/tasks/batch_connect.rake

```diff
@@ -0,0 +1,31 @@
+namespace :batch_connect do
+  desc "Generate new batch connect session"
+  task new_session: :environment do
+    # Read in user settings
+    app = ENV["BC_APP_TOKEN"] || abort("Missing environment variable BC_APP_TOKEN")
+    fmt = ENV["BC_RENDER_FORMAT"]
+    ctx = if !$stdin.tty?
```

Comment:
Why not just:

```ruby
ctx = $stdin.read
ctx = "{}" if ctx.strip.empty?
```

Is there a reason to support multiple ways to run this from the command line? i.e. if I can pipe cat mycontext.json to the command then we don't need BC_SESSION_CONTEXT.

I guess the answer depends on what the reason was for adding this rake task in the first place... (easy testing on other systems?)